operator/cmd/mesh/manifest-apply.go

```diff
@@ -119,10 +132,37 @@ func ApplyManifests(setOverlay []string, inFilenames []string, force bool, dryRu
 	if err != nil {
 		return err
 	}
+	client, err := kubernetes.NewForConfig(kubeconfig)
+	if err != nil {
+		return err
+	}
 	manifests, iop, err := GenManifests(inFilenames, ysf, force, kubeconfig, l)
+
 	if err != nil {
 		return fmt.Errorf("failed to generate manifest: %v", err)
 	}
+
+	dryRunDir := ""
```

Comment:
Pretty sure 'dryRun' usually means don't have any side effects. Maybe you mean 'localMode' ?
Do we really need it ? Please add more comments with the intent.

Reply:
I will remove this part, do not store and generate anything under `druRun`